app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminsController < AuthenticatedController
+  before_filter :ensure_admin
```

Comment:
The new syntax for Rails 4 is `before_action`. They are discouraging `before_filter`